### ISSUE TITLE
 Unable to update extMobilePhone on noridEppContact

### ISSUE THREAD

**Issue by joveice** [metaregistrar/php-epp-client#215]
I'm attempting to update the mobile number on a contact but it doesn't update.

Code sample:
```
$contact = new eppContactHandle($contact_id);
$postal_info = new eppContactPostalInfo($request->name, $request->postal_area, 'NO', $request->org_name, $request->addresses, null, $request->postal_code, eppContact::TYPE_LOC);
$update = new noridEppContact($postal_info, $request->email, $request->phone, null, null, null, $request->type, $request->identity_type, $request->identity_value, $request->mobile, $request->secondary_emails);
$up = new noridEppUpdateContactRequest($contact, null, null, $update);
$response = $conn->request($up));
```
My `$request->mobile` does contain a number. epp logs does not reflect a update, it's giving the old value.
Not sure if this is eppContact or noridEppContact specific so I'm tagging you just in case @alexrsagen since you know the norid module part.

**Comment by alexrsagen** [comment]
I have identified the problem, `$updateInfo` is not being passed to `updateExtContact`, but instead `null` is passed... See https://github.com/metaregistrar/php-epp-client/blob/master/Protocols/EPP/eppExtensions/no-ext-contact-1.0/eppRequests/noridEppUpdateContactRequest.php#L14

I cannot really see a reason for it being the way it is. "My bad", in other words 😉

Fixed in #216.

**Comment by joveice** [comment]
Confirmed.
